app/components/pages/SubmissionWizard/steps/Editors/EditorsContent.js

```diff
@@ -300,39 +301,75 @@ class EditorsContent extends React.Component {
+                  onChange={this.handleSuggestedReviewersChanged}
+                />
+                <ValidatedField
+                  label={
+                    index < limits.suggestedReviewers.min
+                      ? `Reviewer ${index + 1} email`
+                      : `Reviewer ${index + 1} email (optional)`
+                  }
+                  name={`suggestedReviewers.${index}.email`}
+                  onChange={this.handleSuggestedReviewersChanged}
+                  type="email"
+                />
+              </TwoColumnLayout>
             </Box>
           ))}
 
-          {values.opposedReviewers.map((_, index) => (
-            // eslint-disable-next-line react/no-array-index-key
-            <Box key={index} my={3}>
-              <ExcludedReviewer
-                index={index}
-                setFieldValue={setFieldValue}
-                values={values}
+          <OptionalExclude
+            boxVisible={this.isBoxVisible('opposedReviewers')}
+            data-test-id="opposed-reviewers"
+            onRequestClose={() => this.hideBox('opposedReviewers')}
+            onRequestOpen={() => this.showBox('opposedReviewers')}
+            roleName="reviewer"
+          >
+            <FormH3>Exclude Reviewers</FormH3>
+
+            <TwoColumnLayout bottomSpacing={false}>
+              <ValidatedField
+                label="Excluded reviewer 1 name"
+                name="opposedReviewers.0.name"
               />
-            </Box>
-          ))}
+              <ValidatedField
+                label="Excluded reviewer 1 email"
+                name="opposedReviewers.0.email"
+                type="email"
+              />
+            </TwoColumnLayout>
+            <TwoColumnLayout bottomSpacing={false}>
+              <ValidatedField
+                label="Excluded reviewer 2 name"
+                name="opposedReviewers.1.name"
+              />
+              <ValidatedField
+                label="Excluded reviewer 2 email"
+                name="opposedReviewers.1.email"
+                type="email"
+              />
+            </TwoColumnLayout>
 
-          <Box>
-            Would you like to{' '}
-            <MoreButton
-              empty={{ name: '', email: '' }}
-              fieldName="opposedReviewers"
-              objectName="reviewer"
-              setFieldValue={setFieldValue}
-              type="exclude"
-              values={values}
+            <ValidatedField
+              component={Textarea}
+              label="Reason for exclusion"
+              name="opposedReviewersReason"
             />
-            ?
-          </Box>
+          </OptionalExclude>
         </Box>
 
-        <Declaration />
+        <ValidatedField
+          component={ValueCheckbox}
+          label="I declare that, to the best of my knowledge, these experts have no conflict of interest"
+          name="suggestionsConflict"
+        />
       </React.Fragment>
     )
   }
```

Comment:
This file seems to be getting quite large, and the render functions is about 200 lines long. I suggest we pull out some of the variables on the components into arrays of objects and map them onto the components.

e.g.
```
const sections = [
  {
    heading: 'Suggest Senior Editors',
    type: 'include',
    test-id: 'suggested-senior-editors',
    peoplePicker: {
      modal: 'suggestedSeniorEditors',
      selection: 'suggestedSeniorEditors',
      options: seniorEditors,
      title: 'senior editors'
    }
  },
  {
    heading: 'Exclude a Senior Editor',
    type: 'exclude',
    test-id: 'opposed-senior-editors',
    peoplePicker: {
      modal: 'opposedSeniorEditors',
      selection: 'opposedSeniorEditors',
      options: seniorEditors,
      title: 'senior editors'
    }
  },
  ...
]

sections.map((section) => {
  switch(section.type) {
    case 'include':
      <Box>
        <FormH3>{section.heading}</FormH3>
        <Box data-test-id={section.test-id}>
          <PeoplePicker {...section.peoplePicker}
        </Box>
      </Box>
      break;
    case 'exclude':
      <OptionalExclude>
        <FormH3>{section.heading}</FormH3>
        <Box data-test-id={section.test-id}>
          <PeoplePicker {...section.peoplePicker}
        </Box>
      </OptionalExclude>
      break;
  }
});

```

Reply:
Happy to move this onto a seperate issue

Reply:
Completely agreed. Dying to refactor :heart_eyes: but yeah, figured large-scale changes were outside the scope of this issue. Just wanted to get rid of `FormSection.js` as it wasn't really making sense any more.

As for the way in which we refactor, let's have a bit of a chat before raising an issue?